connectors/byoc.py

```diff
@@ -159,6 +159,11 @@ def _create_object(self, doc_source):
         )
 
 
+class SyncJobIndex(ESIndex):
+    def _create_object(self, doc):
+        pass
```

Comment:
This implementation is mandatory

Reply:
It should be done here: https://github.com/elastic/connectors-python/pull/332